core/discovery_test.go

```diff
@@ -19,14 +19,121 @@ under the License.
 
 package core
 
-import "testing"
+import (
+	"strings"
+	"testing"
 
-func TestDiscovery_GetRootNode(t *testing.T) {
-	rootNode, err := GetRootNode()
+	"github.com/hyperledger/fabric/discovery"
+	"github.com/spf13/viper"
+)
+
+func TestDiscovery_GetEmptyRootNode(t *testing.T) {
+	discovery.SetDiscoveryService(NewStaticDiscovery(true))
+	assertRootNode(t, "")
+}
+
+func TestDiscovery_GetEmptyRootNodes(t *testing.T) {
```

Comment:
Won't these empty rootnode tests fail if somebody enters a rootnode in their local config file?